client/layout/logged-out.jsx

```diff
@@ -55,6 +55,8 @@ const LayoutLoggedOut = ( {
 	const localizeUrl = useLocalizeUrl();
 	const isLoggedIn = useSelector( isUserLoggedIn );
 	const currentRoute = useSelector( getCurrentRoute );
+	const pathNameWithoutLocale =
```

Comment:
This approach may result in redirecting to a 404, because in some language we have localized page slugs, e.g.:
`https://wordpress.com/it/support/modifica-css/` → `https://wordpress.com/es/support/modifica-css/` → (404).

On the other hand, prepending the locale slug to the English page slug should redirect correctly:
`https://wordpress.com/support/editing-css/` → `https://wordpress.com/es/support/editing-css/` → `https://wordpress.com/es/support/edicion-de-css/`.

A possible workaround could be to use the pathname from `<link rel="alternate" hreflang="x-default" />` and prepend the locale in `onLanguageChange`, or get the localized URL from `<link rel="alternate" hreflang="LOCALE" />` directly.

Reply:
Wow, this scenario totally went over my head.

This code in this particular comment is alright, I think. Because it only runs in Calypso. All locales have the same pathnames. But the point totally stands in the support site. I'll fix.